gtsfm/evaluation/eval_hilti.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python3
+
+# Created by Beda Berner
+# beda.berner@hilti.com
+
+# Modified for using both fastlio and gtsfm TUM files
+# If you already have matplotlib, scipy and numpy, this will need an additional
+# pip install evo==1.13.5
```

Comment:
Should eval_hilti script be in master branch?

Reply:
I am not sure. I originally put in it in the evaluation directory above gtsfm/ directory, which is currently empty, because it is not part of gtsfm. 

Or we could also move it to 3rdparty.

Reply:
I am note sure about this. cc @dellaert

Reply:
This is a pending comment

Reply:
if we are not going to use this often, no need to depend on evo. Feel free to mark this resolved.